apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -165,6 +174,39 @@ export function generateReactHooks<
         }, [rowAction, onComplete])
     }
 
+    function useCount (variables: QueryVariables, options?: QueryHookOptions<IUseCountQueryReturnType, QueryVariables>) {
+        const intl = useIntl()
+        const AccessDeniedError = intl.formatMessage({ id: 'AccessError' })
+        const ServerError = intl.formatMessage({ id: 'ServerErrorPleaseTryAgainLater' })
+
+        const { data, error, loading, refetch, fetchMore, stopPolling } = useQuery<IUseObjectsQueryReturnType<GQLObject>, QueryVariables>(gql.GET_COUNT_OBJS_QUERY, {
+            variables,
+            notifyOnNetworkStatusChange: true,
```

Comment:
Should we add it to others heeks

Reply:
This is already used in other hooks in generate.hooks